mark_outliers_in_json.py

```diff
@@ -6,15 +6,16 @@
 usage: Write lists of outliers into Krun results file(s).
 Example usage:
 
-$ python mark_outliers_in_json.py -f results1.json.bz2
-$ python mark_outliers_in_json.py -f results1.json.bz2 -f results2.json.bz2 --window 250
-       [-h] [--filename JSON_FILES] [--window WINDOW_SIZE]
+$ python mark_outliers_in_json.py results1.json.bz2
+$ python mark_outliers_in_json.py ---window 250 results1.json.bz2 results2.json.bz2
+        [-h] [--window WINDOW_SIZE] [--threshold THRESHOLD] json_files
```

Comment:
Help text is automatically generated by the `argparse` module, this file-level documentation is copied from that